File: M04_CHRYSANTHEMUM/oneGame/.resbackup-0/lua/races.lua
require( "default_races" );

-- Returns a table with the race values
function getRaceValues ( raceIndex )
	if ( raceIndex == nil ) then
		print( "Bad race index!" );
	end
	raceTable = RACES.races[raceIndex];
	return raceTable;	-- BOOM
end

File: N03_CLOUDCOMPUTING/oneGame/.res-0/lua/races.js
core_require( "default_races/races.js" );

// Returns a table with the race values
function getRaceValues ( raceIndex )
{
	if ( !raceIndex ) {
		print( "Bad race index!" );
	}
	raceTable = RACES.races[raceIndex];
	return raceTable; // sexy
}